client/src/modules/MentorRegistry/components/MentorRegistryTableContainer.tsx

```diff
@@ -337,17 +384,51 @@ export const MentorRegistryTableContainer = ({
   };
 
   const handleClearAllButtonClick = () => {
-    setCombinedFilter({ preferredCourses: [], technicalMentoring: [], preselectedCourses: [] });
+    setCombinedFilter({
+      preferredCourses: [],
+      technicalMentoring: [],
+      preselectedCourses: [],
+      githubId: [],
+      cityName: [],
```

Comment:
```suggestion
    preferredCourses: string[];
    technicalMentoring: string[];
    preselectedCourses: string[];
    githubId: string[];
    cityName: string[];
```

Reply:
looks strange as it's passing props to function but not type declaration

Reply:
my bad